src/Middleware/HttpsPolicy/src/HttpsRedirectionMiddleware.cs

```diff
@@ -26,7 +29,7 @@ public class HttpsRedirectionMiddleware
         private readonly ILogger _logger;
 
         /// <summary>
-        /// Initializes the HttpsRedirectionMiddleware
+        /// Initializes the HttpsRedirectionMiddleware.
```

Comment:
```suggestion
        /// Initializes the <see cref="HttpsRedirectionMiddleware" />.
```

Since we're here already, might as well put a `<see />` in.